src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -119,9 +145,16 @@ export default function ActionsPanel(): JSX.Element {
             setActionData(actions.results[0].actions);
           }
           setFetchingActionData(false);
+          setInlineErrors((prevInlineErrors) => [null, prevInlineErrors[1]]);
```

Comment:
This case (and the other setInlineErrors() calls below) would lead to bugs if the total number of errors in the tuple changed (e.g. if a third error was added it would get lost during this call). Instead we should only update the slot that matches the error we're concerned with.

If you're concerned about mutating state then you could do something like:
```
errors = [... prevInlineErrors];
errors[0] = null;
```